openedx/core/djangoapps/content/course_overviews/tests/test_course_overviews.py

```diff
@@ -356,10 +356,10 @@ def test_nonexistent_course_does_not_exists(self, modulestore_type):
 
     def test_get_errored_course(self):
         """
-        Test that getting an ErrorDescriptor back from the module store causes
+        Test that getting an ErrorBlock back from the module store causes
         load_from_module_store to raise an IOError.
         """
-        mock_get_course = mock.Mock(return_value=ErrorDescriptor)
+        mock_get_course = mock.Mock(return_value=ErrorBlock)
         with mock.patch('xmodule.modulestore.mixed.MixedModuleStore.get_course', mock_get_course):
             # This mock makes it so when the module store tries to load course data,
             # an exception is thrown, which causes get_course to return an ErrorDescriptor,
```

Comment:
```suggestion
            # an exception is thrown, which causes get_course to return an ErrorBlock,
```

Reply:
I am planning to do a search-and-replace through all the code for the Descriptor/Module terminology after all the conversions so not super important to update everything right now.